airflow/www/api/experimental/endpoints.py

```diff
@@ -35,7 +36,17 @@
 
 log = logging.getLogger(__name__)
 
-requires_authentication = airflow.api.API_AUTH.api_auth.requires_authentication
```

Comment:
This is a nightmare because `airflow.api.API_AUTH.api_auth` variable is empty by default. We can't load this module without calling another method beforehand.